src/js/controller/viewMixin/core.js

```diff
@@ -156,7 +156,7 @@ var Core = {
      * @param {function} [iteratee] - iteratee function invoke each view models
      */
     positionViewModels: function(start, end, matrices, iteratee) {
-        var ymdListToRender, endTime;
+        var ymdListToRender, endTime, isSameDate;
```

Comment:
`isSameDate` variable is unnecessary to be declared here. Instead, It's better a line 171.